app/services/shopify_import/creators/base.rb

```diff
@@ -0,0 +1,19 @@
+module ShopifyImport
+  module Creators
+    class Base
+      def initialize(shopify_data_feed)
+        @shopify_data_feed = shopify_data_feed
+      end
+
+      def save!
+        raise NotImplementedError
+      end
+
+      private
+
+      def data_feed_value
```

Comment:
I'd call this method just `data_feed`. `data_feed_value` sounds a bit confusing to me as I'd expect it to have a single value like some kind of status ("active", "pending", etc.) or a number (1..5)